### ISSUE TITLE
7.0 upgrade to 7.2

### ISSUE THREAD

**Issue by cverymuch** [PowerShell/PSReadLine#2543]
### Environment
PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0-preview.6
OS: Microsoft Windows 10.0.19041
BufferWidth: 120
BufferHeight: 9001

Last 0 Keys
```

```

### Exception
```
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset() in Microsoft.PowerShell.PSReadLine2.dll:token 0x60004a6+0x0
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset() in Microsoft.PowerShell.PSReadLine2.dll:token 0x6000478+0x7
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics) in Microsoft.PowerShell.PSReadLine2.dll:token 0x60000f2+0x118
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken) in Microsoft.PowerShell.PSReadLine2.dll:token 0x60000eb+0x60
```

**Comment by A55imilat0r** [comment]
Yep I'm getting this as well on latest Win 10 insider build 

### Environment
PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0-preview.6
OS: Microsoft Windows 10.0.21390
BufferWidth: 120
BufferHeight: 30

Last 0 Keys
```

```

### Exception
```
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset() in Microsoft.PowerShell.PSReadLine2.dll:token 0x60004a6+0x0
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset() in Microsoft.PowerShell.PSReadLine2.dll:token 0x6000478+0x7
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics) in Microsoft.PowerShell.PSReadLine2.dll:token 0x60000f2+0x118
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken) in Microsoft.PowerShell.PSReadLine2.dll:token 0x60000eb+0x60
```

**Comment by jpomfret** [comment]
I was having the same issues - it's due to a breaking changes introduced in PowerShell 7.2.0-preview.6.

Fix is to update PSReadLine to [2.2.0-beta3](https://github.com/PowerShell/PSReadLine/releases/tag/v2.2.0-beta3)

Details on the PR: https://github.com/PowerShell/PSReadLine/pull/2524
